protocols/krausehouse/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "krausehouse",
+  "description": "Krause House is a DAO empowering fans to purchase and operate an NBA team.",
+  "path": "krausehouse",
+  "type": "snapshot",
+  "isEnabled": true,
+  "treasuryAddress": ["0x314e65DE454D78D4F685545824448371FeF4aac3"]
```

Comment:
not very confident about this one. waiting to hear back from kraus haus.

Reply:
No worries :) Let's wait until we get a confirmation

Reply:
Updated with the one that Tangfeng shared.